### PR TITLE
login pg

### PR DESCRIPTION
**Pull request by aurobindoGupta**
image

### CODE REVIEW

File: components/loginPg/index.html [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+      <div class="navBar-searchBar">
+        <form class="searchBar">
+          <input class="searchBar-input" placeholder="Search" />
+        </form>
+      </div>
+      <ul class="navLinks">
+        <li class="navLinksBtn">
+          <button class="btn btn-secondary">Login</button>
+        </li>
+        <li class="navLinksBtn" onclick="window.location.href=''">
```
Comment (Aditi0505) [inline components/loginPg/index.html]: Please remove onClick if not needed. Or use a JavaScript file separately.

File: components/loginPg/index.html [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+          <div class="ic-badge container">
+            <i class="fa fa-heart ic"></i>
+            <div class="iconBadge">0</div>
+          </div>
+        </li>
+        <li class="navLinksBtn" onclick="window.location.href=''">
```
Comment (Aditi0505) [inline components/loginPg/index.html]: Please remove onClick if not needed. Or use a JavaScript file separately.

File: components/loginPg/index.html [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+    <!-- ................NAV BAR............. -->
+    <!-- ................PAGE CONTENT ............................... -->
+
+    <div class="page-content">
+      <div class="login-Modal">
+        <div class="modal-container">
+          <div class="modal-title">
+            <p class="fw-bold fs-L">Login</p>
+          </div>
+          <div class="form-input">
```
Comment (Aditi0505) [inline components/loginPg/index.html:60]: Can use form here instead of a div

**Review by Aditi0505 — APPROVED** [review]
Left few suggestions. Rest looks good.

File: components/loginPg/index.html (hunk 1 above)
Comment (aurobindoGupta) [reply components/loginPg/index.html]: done

**Review by akaysing — APPROVED** [review]
looks good to me

**Pull request merged**